modules/component/migrations/14_1_0/index.spec.ts

```diff
@@ -0,0 +1,77 @@
+import { Tree } from '@angular-devkit/schematics';
+import {
+  SchematicTestRunner,
+  UnitTestTree,
+} from '@angular-devkit/schematics/testing';
+import * as path from 'path';
+import { createPackageJson } from '@ngrx/schematics-core/testing/create-package';
+import { waitForAsync } from '@angular/core/testing';
+
+describe('Component Store Migration 14_1_0', () => {
+  let appTree: UnitTestTree;
+  const collectionPath = path.join(__dirname, '../migration.json');
+  const pkgName = 'component';
+
+  beforeEach(() => {
+    appTree = new UnitTestTree(Tree.empty());
+    appTree.create(
+      '/tsconfig.json',
+      `
+        {
+          "include": [**./*.ts"]
+        }
+       `
+    );
+    createPackageJson('', pkgName, appTree);
+  });
+
+  describe('Replace ReactiveComponentModule', () => {
+    it(
+      `should replace the ReactiveComponentModule with LetModule and PushModule`,
+      waitForAsync(async () => {
+        const input = `
+      import { ReactiveComponentModule } from '@ngrx/component';
+
+      const reactiveComponentModule: ReactiveComponentModule;
+
+      @NgModule({
+        imports: [
+          AuthModule,
+          AppRoutingModule,
+          ReactiveComponentModule,
+          CoreModule,
+        ],
+        bootstrap: [AppComponent],
+      })
+      export class AppModule {}
+    `;
+        const expected = `
+      import { LetModule, PushModule } from '@ngrx/component';
+
+      const reactiveComponentModule: ReactiveComponentModule;
```

Comment:
I'm not migrating assignments to `ReactiveComponentModule`. However, currently I am not preserving the import. This PR could be enhanced by preserving the import of `ReactiveComponentModule` when it has not been replaced in the file.

Reply:
I think it should be preserved in this case. 
A migration shouldn't break an application (Ideally 😅).
If it's hard to do that, I would prefer to keep the `ReactiveComponentModule` import for all cases.